package/base-files/files/etc/init.d/led

```diff
@@ -72,6 +72,8 @@ load_led() {
 			[ -n "$dev" ] && {
 				echo $dev > /sys/class/leds/${sysfs}/device_name
 				echo $mode > /sys/class/leds/${sysfs}/mode
+				[ -n "$interval" ] && \
```

Comment:
Please remove the `[ -n "$interval" ] && \` condition. Due to the `config_get interval $1 interval "50"` at the beginning of the the `load_led()` function, `$interval` is set to 50 if not defined,